packages/dd-trace/src/lambda/runtime/ritm.js

```diff
@@ -87,7 +87,7 @@ const registerLambdaHook = () => {
   const lambdaTaskRoot = process.env.LAMBDA_TASK_ROOT
   const originalLambdaHandler = process.env.DD_LAMBDA_HANDLER
 
-  if (originalLambdaHandler !== undefined) {
+  if (originalLambdaHandler !== undefined && lambdaTaskRoot !== undefined) {
```

Comment:
Should we add a comment mentioning the SST requirement?

Reply:
I don't think so because I think most emulators don't set this env var, so it's a good thing to check generally.